### PR TITLE
support delete multi env

### LINKED ISSUE #313: [Feature] `vela env delete` should be able to delete multiple env
```
$ vela env ls
NAME         	CURRENT	NAMESPACE
default      	    *   	default
env-e2e-hello	       	env-e2e-hello
test         	       	default
```

I hope to delete two env at one time like this:

```
$ vela env delete env-e2e-hello test
env-e2e-hello deleted
```

But now, only one env is deleted.

```
vela env ls
NAME   	CURRENT	NAMESPACE
default	  *     	default
test   	       	default
```

### PR DESCRIPTION
**Pull request by silenceper**
fix #313 ,support delete multi env

### CODE REVIEW

**Review by wonderflow — APPROVED** [review]
LGTM， thanks!

File: pkg/commands/env.go [hunk 1]
```diff
@@ -137,12 +137,14 @@ func DeleteEnv(ctx context.Context, args []string, ioStreams cmdutil.IOStreams)
...
 	}
-	envName := args[0]
-	msg, err := oam.DeleteEnv(envName)
-	if err == nil {
+	for _, envName := range args {
+		msg, err := oam.DeleteEnv(envName)
+		if err != nil {
+			return err
+		}
 		ioStreams.Info(msg)
```
Comment (wonderflow) [inline pkg/commands/env.go:145]: would you mind to refine the output message？It will output multiple times now

Comment (silenceper) [reply pkg/commands/env.go:145]: You mean to display it in one line?  like:
`test1, test2 deleted `

I see that kubectl also displays multiple lines:
```
pod "xxx" deleted
pod "xxxx" deleted
```

Is it necessary to merge together？

Comment (silenceper) [reply pkg/commands/env.go:145]: Do you mean to combine and output? like 
```
test1,test2 deleted.
```

Comment (wonderflow) [reply pkg/commands/env.go:145]: yeah, it's fine

Comment (silenceper) [reply pkg/commands/env.go:145]: Is it necessary? I see that the kubectl command is also multi-line output, like:
```
pod "pod-1" deleted
pod "pod-2" deleted
```

Of course I can also change the output into one line?

Comment (wonderflow) [reply pkg/commands/env.go:145]: Well, it also makes sense, let me merge, thanks again !

**Pull request merged**